_posts/2018-11-16-stairway-to-health.md

```diff
@@ -0,0 +1,94 @@
+We developed the dashboard working around some key questions while keeping the appearance of the dashboard in line with the dashboard JWorks developed.
+
+In the following section I will explain how the dashboard is currently set up and how to use it properly.
+Next I will go over the features we can add in the future to allow the user to go even deeper in their analysis.
+
+# Dashboard overview
+
+The dashboard is built to answer **the following questions**:
+
+1. What is the percentage of people taking the stair/elevator at Ordina **today**?
+2. How is the same metric on week/month/year basis? What is it in absolute numbers?
+3. Within each day/week/month/year base how does it evolve over time?
+4. And across days/weeks/months/years? Are people taking the stairs more this week compared to last week?
+
+To answer these questions, the dashboard consists of **three main parts**.
+I will go over these parts and highlight which question(s) it tries to answer.
+
+## Part one: the Title
+
+The title is what the user sees first and answers the first question.
+By using the colors in the title, the dashboard shows the user - in a subtle way - what the colors in the next visuals represent.
+There is also an option to select another day as illustrated below.
+
+![PartOne](/img/2018-11-16-stairway-to-health/PartOne.gif)
+
+## Part two: the horizontal bar comparison
+
+In part two the user can find an answer to questions two and three.
+The visual uses the selected day to show the division between people taking the stair / elevator on a daily / weekly / monthly and yearly basis.
+When the user hovers over the chart he can also see the evolution of people taking the stair / elevator within that day / week / month / year.
+Next to the chart, the total absolute number of all the observations measured is reported per period.
+
+![PartTwo](/img/2018-11-16-stairway-to-health/PartTwo.gif)
+
+## Part three: the more detailed area chart
+
+The third part visualises how the division stair / elevator is evolving over time expressed in day / week / month or year basis.
+This gives the user the possibility to look at trends and to see how the situation of today compares itself to past situations.
+
+In the title the user has the option to change the appearance of the data (absolute or shares).
+He has also the option to change how many periods are visualised starting from the most recent period.
+
+![PartThreeOne](/img/2018-11-16-stairway-to-health/PartThreeOne.gif)
+
+When the user hovers over the chart he can see the same horizontal bar comparison allowing him to compare the selected period above with the period he is hovering over in the chart.
```

Comment:
Proposed rephrase:

When the user hovers over the chart the same horizontal bar comparison can be seen, comparisons can be made with the period selected above.